content/en/mobile/_index.md

```diff
@@ -48,19 +48,39 @@ On the Dashboards page, you can view and search all of the dashboards that you h
 
 **Note:** To set up or edit a dashboard, you need to [login to the Datadog browser app][9]. For more information, see [Dashboards][5]. 
 
-<!--- ## Quick Actions  --->
+## Quick Actions
 
-<!--- {{< img src="mobile/quick_actions.png" alt="dashboard page">}}  --->
+{{< img src="mobile/quick-actions-3.png" alt="Quick Actions">}}
 
-<!--- Long-press on your Datadog app icon to surface a quick-action sheet of your top dashboards. On iOS, it shows your top five “Frequently Viewed By Me” preset dashboards. On Android, it shows your top five viewed dashboards on mobile.  --->
+Long-press on the app icon to surface a quick-action sheet of your top five [Frequently Viewed By Me][10] dashboards for iOS (measured by view count and recency) and your five most opened dashboards on mobile for Android. Press on a result to open the dashboard in-app.
+
+## Home Screen Search
+
+{{< img src="mobile/home-screen-search-2.png" alt="Home Screen Search" style="width:40%;">}}
+
+**iOS only**: Within iPhone Search, filter and search for the name of any desired dashboard. Press on a result to open the dashboard view directly on the mobile app, or press the “Search in App” button to open the search query in the in-app Dashboard List page.
+
+## Siri Shortcuts and Suggestions
+
+**iOS Only**: You can create Siri Shortcuts for Datadog dashboards and monitors. To do so, create a shortcut through the Shortcuts App. For a shortcut to be available to create, you must first execute the desired action at least once in the app. For example, to create an “Open Mobile App Screenboard” shortcut, you must first open the Mobile App Screenboard in your mobile app at least once.
+
+With the shortcut, you can access your dashboards and monitors through three key actions:
+
+* Pin the shortcut as an icon in your home screen. To do so, access the Shortcuts app, and open the edit menu for your dashboard shortcut.
+* Siri Voice: say your shortcut name, such as “Open HAproxy and Bunk” in the image below.
```

Comment:
Change to:
Siri Voice: say your shortcut name, such as “Open AWS Overview”, and Siri will open your dashboard in-app.
Siri suggestions: Siri learns your routine, and suggests dashboard shortcuts when you most need them, through a home or lock-screen banner, iPhone search, or iOS 14 Siri Suggestions widget.